<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>东航金融仓单回购系统报价终端</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Administrator/.designer/backup/Title.png</normaloff>C:/Users/Administrator/.designer/backup/Title.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_title" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>33</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="lab_Ico">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="WrUiClient.qrc">:/image/logo.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_Title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>东航金融仓单综合交易系统报价终端</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_menu" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnMenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>菜单</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMenu_Min">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>最小化</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMenu_Max">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>最大化</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMenu_Close">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>关闭</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="MainWindow_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_syms">
               <property name="text">
                <string>合约列表</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="update_symlst_btn_">
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sub_btn_">
               <property name="text">
                <string>订阅所有</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unsub_btn_">
               <property name="text">
                <string>退订所有</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="syms_lst_view_"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="8" column="3">
           <widget class="QLabel" name="m_DiscountPrice">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="label10">
            <property name="text">
             <string>到期逆回购率</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QLabel" name="label16">
            <property name="text">
             <string>第一期货</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>品种</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label4">
            <property name="text">
             <string>回购日</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="m_DiscountRatio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label9">
            <property name="text">
             <string>正回购率</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="m_PosiRepoRatio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label11">
            <property name="text">
             <string>贴现价</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QLabel" name="m_ReveRepoRatio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="m_ExchangeId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>贴现/回购天数</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label8">
            <property name="text">
             <string>贴现率</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>交易所</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4">
           <widget class="QLabel" name="label17">
            <property name="text">
             <string>第一期货价</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QLabel" name="label12">
            <property name="text">
             <string>正回购价</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3">
           <widget class="QLabel" name="m_PosiRepoPrice">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="m_ContractDate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="5">
           <widget class="QLabel" name="m_AlphaLastPrice">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="m_ContractDays">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label15">
            <property name="text">
             <string>有效交割日</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="m_DeliveryDate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="m_ProductId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QLabel" name="m_AlphaInstrumentId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_quote">
            <property name="text">
             <string>查询报价</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QLabel" name="label14">
            <property name="text">
             <string>灵活逆回购截止日</string>
            </property>
           </widget>
          </item>
          <item row="5" column="6">
           <widget class="QLabel" name="label6">
            <property name="text">
             <string>灵活逆回购率</string>
            </property>
           </widget>
          </item>
          <item row="7" column="6">
           <widget class="QLabel" name="label7">
            <property name="text">
             <string>灵活逆回购价</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QLabel" name="m_DeadlineDate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QLabel" name="label13">
            <property name="text">
             <string>第二期货</string>
            </property>
           </widget>
          </item>
          <item row="8" column="5">
           <widget class="QLabel" name="m_BravoInstrumentId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="7">
           <widget class="QLabel" name="m_FlexReveRepoPrice">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="7">
           <widget class="QLabel" name="m_FlexReveRepoRatio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="4">
           <widget class="QLabel" name="label5">
            <property name="text">
             <string>第二期货价</string>
            </property>
           </widget>
          </item>
          <item row="9" column="5">
           <widget class="QLabel" name="m_BravoLastPrice">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="6">
           <widget class="QLabel" name="label0">
            <property name="text">
             <string>更新日期</string>
            </property>
           </widget>
          </item>
          <item row="8" column="7">
           <widget class="QLabel" name="m_SystemDate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="6">
           <widget class="QLabel" name="label18">
            <property name="text">
             <string>更新时间</string>
            </property>
           </widget>
          </item>
          <item row="9" column="7">
           <widget class="QLabel" name="m_SystemTime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="Line" name="line2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_sub">
            <property name="text">
             <string>订阅报价</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="sub_tbl_view_"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar2"/>
 </widget>
 <resources>
  <include location="WrUiClient.qrc"/>
 </resources>
 <connections/>
</ui>
